delfin/task_manager/scheduler/schedulers/telemetry/telemetry_job.py

```diff
@@ -96,4 +100,6 @@ def _schedule_failed_telemetry_job_handler(self, ctx):
             FailedTelemetryJob(ctx), 'interval',
             seconds=TelemetryCollection.FAILED_JOB_SCHEDULE_INTERVAL,
             next_run_time=datetime.now(),
-            id=periodic_scheduler_job_id)
+            id=periodic_scheduler_job_id,
+            misfire_grace_time=int(
+                CONF.telemetry.performance_collection_interval / 2))
```

Comment:
Can you just describe the impact of passing this parameter

Reply:
If we do not pass the para, some task would not be executed because of timeout, details in https://github.com/sodafoundation/delfin/issues/603